.github/workflows/pytest.yml

```diff
@@ -54,8 +54,11 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-pip-
 
+      # sphinxcontrib-applehelp added due to https://github.com/coddingtonbear/python-measurement/issues/75.
```

Comment:
Would it be good to create an issue on our side, and link it with the related GH issue? This give us better way of tracking this, instead of placing this note in the .yml file.

Reply:
https://github.com/saleor/saleor/issues/11664